### ISSUE TITLE
Le paramétrage du test de Kolmogorov change avec ot 1.12

### ISSUE THREAD

**Issue by adumasphi** [persalys/persalys#121]
In GitLab by @schueller on Sep 17, 2018, 09:24

Il faudra changer 1-level=>level:
```
lib/src/base/InferenceAnalysis.cxx:203:        TestResult testResult(FittingTest::Kolmogorov(sample.getMarginal(i), distribution, 1 - level_, distribution.getParameterDimension()));
```

cf https://github.com/openturns/openturns/pull/829

**Comment by adumasphi** [comment]
In GitLab by @schueller on Oct 4, 2018, 09:24

mentioned in commit 9c9931a91bec18d3be31691756e599ee96cc7a10

**Comment by adumasphi** [comment]
In GitLab by @schueller on Oct 5, 2018, 18:07

mentioned in commit f3083781d73da4b526a8ea56f75de5e8e8d16b69

**Comment by adumasphi** [comment]
In GitLab by @schueller on Oct 5, 2018, 18:07

closed via commit 9c9931a91bec18d3be31691756e599ee96cc7a10

**Comment by adumasphi** [comment]
In GitLab by @schueller on Oct 5, 2018, 18:07

closed via merge request !209
